functions.php

```diff
@@ -87,6 +87,12 @@ function twentyseventeen_setup() {
 		'flex-width'  => true,
 		'flex-height' => true,
 	) );
+
+	/*
+	 * This theme styles the visual editor to resemble the theme style,
+	 * specifically font, colors, and column width.
+ 	 */
+	add_editor_style( array( 'assets/css/editor-style.css', twentyseventeen_fonts_url() ) );
 }
 endif;
 add_action( 'after_setup_theme', 'twentyseventeen_setup' );
```

Comment:
Thanks for the code review, @ntwb!

Everything should be addressed, except this one - it looks like both the functions.php and the editor-style.css files already already end with a new line in this commit. I might be misunderstanding - please let me know if this is what you meant, or something else. Thanks! 🙂